### ISSUE TITLE
markers helper can't detect markers which are separated with space.

### ISSUE THREAD

**Issue by sivchari** [kubernetes-sigs/kube-api-linter#53]
https://github.com/kubernetes-sigs/cluster-api/blob/main/api/v1beta2/cluster_types.go#L1285-L1296

```go
// +kubebuilder:object:root=true

// ClusterList contains a list of Cluster.
type ClusterList struct {
	metav1.TypeMeta `json:",inline"`
	// metadata is the standard list's metadata.
	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#lists-and-simple-kinds
	// +optional
	metav1.ListMeta `json:"metadata,omitempty"`
	// items is the list of Clusters.
	Items []Cluster `json:"items"`
}
```

Above code must be reported by statussubresource linter since this custom resource has root marker and must also have status marker.
But currently marker accessor helper can't collect it because there is space between root marker and ClusterList type.

So I think we should discuss about 2 topic.

1. The way to be able to collect all markers in spite of space.  
2. Configuration for statussubresource. (e.g. ignore error when the type has List suffix on itself.)

**Comment by sivchari** [comment]
we should handle second level comment.

/remove lifecycle stale

**Comment by sivchari** [comment]
Hmm, surely the statussubresource linting seemed to become more strict, but the second level comment bug is still valid.
So I reopen this issue.

/reopen

**Comment by saschagrunert** [comment]
> Hmm, surely the statussubresource linting seemed to become more strict, but the second level comment bug is still valid. So I reopen this issue.

True, let me address this issue in https://github.com/kubernetes-sigs/kube-api-linter/pull/187

**Comment by JoelSpeed** [comment]
I certainly would have expected that to pick up the disconnected markers at this point. @saschagrunert wild guess but, is it possible that the logic for "too far away" is counting the first line in the comment, not the last?

We should add this field as a test case and debug 👀

**Comment by rosenhouse** [comment]
Awesome, can confirm it's now working as I expect, thank you!!
